### DISCUSSION TITLE
CAT wallet should retain automatically showing new CAT's

### DISCUSSION THREAD

**Discussion by DrKiz** [Chia-Network/chia-blockchain#10236] [Ideas]
I see that the new beta 1.3 of the lite wallet doesn't show new cat's in the wallet unless you enter the tails id.

It was great when these were automatically identified, all you had to do was look up the cat id from the tails database and rename and you were good to go.

Now a person needs to keep a running list on the side for any CATS he knows he has, and will never know that they were sent random CATS which was a bonus before, and is no longer now since you removed the feature of auto id cats.

**Comment by trgarrett** [comment]
I agree. If the idea is to speed up the sync, allowing users to turn off sync (and maintain a separate sync status) per CAT wallet would seem to be a lot more user friendly. Another variation would be closing out your holdings on a particular CAT you’d moved away from. Being able to stop sync and hide the wallet would be HUGE.

**Comment by ForkFarmer** [comment]
Chia 1.3.4 fixes this:

CLI: chia rpc wallet get_stray_cats

or add to config to for them to show up in GUI
automatically_add_unknown_cats: true
